pom.xml

```diff
@@ -6,42 +6,46 @@
     <parent>
         <groupId>uk.gov.companieshouse</groupId>
         <artifactId>companies-house-parent</artifactId>
-        <version>1.3.0</version>
+        <version>2.1.9</version>
         <relativePath/>
     </parent>
 
     <artifactId>document-signing-api</artifactId>
     <version>unversioned</version>
     <name>document-signing-api</name>
     <description>API to sign documents digitally.</description>
-	<packaging>jar</packaging>
+    <packaging>jar</packaging>
+
 
     <properties>
-        <java.version>11</java.version>
-        <spring-boot-dependencies.version>2.7.8</spring-boot-dependencies.version>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <!-- SpringBoot/Maven Dependencies-->
+        <java.version>21</java.version>
+        <spring-boot-dependencies.version>3.3.5</spring-boot-dependencies.version>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <maven-surefire-plugin.version>3.5.2</maven-surefire-plugin.version>
+        <spring-boot-maven-plugin.version>3.3.5</spring-boot-maven-plugin.version>
+        <commons-io.version>2.17.0</commons-io.version>
+        <!-- CH Dependencies-->
         <start-class>uk.gov.companieshouse.documentsigningapi.DocumentSigningApiApplication</start-class>
-
-        <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-
+        <api-security-java.version>2.0.8</api-security-java.version>
+        <structured-logging.version>3.0.20</structured-logging.version>
+        <environment-reader-library.version>3.0.1</environment-reader-library.version>
+        <!-- Testing Dependencies-->
+        <testcontainers.version>1.20.3</testcontainers.version>
+        <mockito-core.version>5.14.2</mockito-core.version>
+        <system-stubs-version>2.1.7</system-stubs-version>
+        <junit.version>4.13.2</junit.version>
```

Comment:
Should be migrated to https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api